curvefs/conf/client.conf

```diff
@@ -1,11 +1,12 @@
 # mdsOpt
-mdsOpt.mdsMaxRetryMS=8000
+mdsOpt.mdsMaxRetryMS=16000
```

Comment:
// RPC total retry time with MDS
mdsOpt.mdsMaxRetryMS=16000
//The maximum timeout of RPC communicating with MDS. 
//The timeout of exponential backoff cannot exceed this value
mdsOpt.rpcRetryOpt.maxRPCTimeoutMS=2000
//TODO(chengyi): after curvefs-tools support create fs, this value can lower down to 500
//RPC timeout for once communication with MDS
mdsOpt.rpcRetryOpt.rpcTimeoutMs=5000
//RPC with mds needs to sleep for a period of time before each retry
mdsOpt.rpcRetryOpt.rpcRetryIntervalUS=50000
//Switch if the number of consecutive retries on the current MDS exceeds the limit. 
//The number of failures includes the number of timeout retries
mdsOpt.rpcRetryOpt.maxFailedTimesBeforeChangeAddr=3
//The normal retry times for trigger wait strategy
mdsOpt.rpcRetryOpt.normalRetryTimesBeforeTriggerWait=3
//Sleep interval for wait
mdsOpt.rpcRetryOpt.waitSleepMs=1000